### DISCUSSION TITLE
SEB Server integrations with all the supported LMS systems

### DISCUSSION THREAD

**Discussion by imrhlrvndrn** [SafeExamBrowser/seb-server#104] [Q&A]
Hey @anhefti, Could you shed some light on all the available guidance on setting up all the supported Learning Management Systems (LMS) with SEB Server.

Since the `Start URL` approach is not supported in the current version of SEB we're looking into integrating SEB Server with various LMS platforms, and I would greatly appreciate your assistance in providing clear instructions or documentation on how to achieve this seamlessly.

Specifically, I am interested in understanding the steps involved in configuring SEB Server to work seamlessly with the supported LMS options. Any information, guidelines, or best practices you can share would be immensely helpful.

If there are specific resources, documentation, or tutorials available, please direct me to them. Additionally, if there are any updates or considerations that have been introduced in recent releases of SEB Server, I would appreciate being made aware of those as well.

Thank you very much for your time and assistance. I look forward to your guidance on successfully setting up SEB Server with various LMS platforms.

**Comment by anhefti** [comment]
Hi, 
Please read the user manual for [SEB Server about LMS Setups](https://seb-server.readthedocs.io/en/latest/lmssetup.html).
